frontend/src/Components/HomePage/AddTrip/FlightForm.jsx

```diff
@@ -109,15 +88,14 @@ const FlightForm= () => {
             return;
         }
         
-        if (!flightStatus || !flightStatus.data || flightStatus.data.length === 0) {
+        if (!flightData === 0) {
```

Comment:
`flightData !== 0` though I would prefer seeing it to null if it doesn't exist and using numerical check if you're looking at the length of an array